builders/manage/dapp/xcm-channels.md

```diff
@@ -0,0 +1,101 @@
+
+1. Select the **registrar** storage module
+2. Select **registrarDeposit**
+3. Insert your appchain id
+4. Press **+** icon
+
+You'll see the registration account at the bottom.
+
+![Locating your registration account](/images/builders/manage/dapp/xcm-channels/xcm-channels-1.webp)
+
+### Retrieving the Sudo Account {: #retrieving-sudo-account }
+
+If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/dapp/xcm-channels/xcm-channels-2.webp)
+
+!!! warning
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Accessing the Channel Management Panel {: #accesing-channel-management-panel }
+
+The Tanssi dApp makes it easy for the appchain governor to manage cross-chain communication channels. It allows requesting new channels, accepting incoming channel opening requests, closing existing channels, and more. To access the channel management panel, head to the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}, click on the **Manage** button, and then, again click on the **XCM** button.
+
+![Accessing the channel management panel](/images/builders/manage/dapp/xcm-channels/xcm-channels-3.webp)
+
+!!! note
+    If you don't see your appchain's details on the dashboard, make sure to comply with the [prerequisites](#checking-prerequisites).
+
+The panel will show your appchain's communication channels status along with several available actions. The elements you are presented with are:
+
+- Sovereign account
+- Incoming/Outgoing channel requests - 
+- Established channels
+- Request to open new channels
+
+![The channel management panel](/images/builders/manage/dapp/xcm-channels/xcm-channels-4.webp)
+
+### Sovereign Account {: #sovereign-account }
+
+Before opening a new channel, the appchain's sovereign account in the relay chain must be funded with enough tokens to be locked as a channel deposit.
+
+A sovereign account is a keyless account that belongs to the appchain in a different consensus system, the relay chain in this case. It can only be used by the appchain's governor.
+
+In this section, you can see your appchain's sovereign account balance, copy its address, and deposit tokens.
+
+!!! note
+    Your appchain's sovereign account in the relay chain must be funded with relay chain tokens. On Dancebox, the relay chain uses UNIT tokens, which the Tanssi team will help you obtain when needed.
```

Comment:
how can they get ahold of the team?

Reply:
I changed this wording a bit, now it says "[...] On Dancebox, the relay chain uses UNIT tokens, and the required amount is currently configured to zero."